### ISSUE TITLE
[Feb 2025] Accessibility Audit: Patients page flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8512]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the patient management user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/patient`
* `/app/patients`
* `/app/add-patient`

Use cases to test:
* view patients list with no patients and multiple patients
* archive patient modal
* add a patient
* edit a patient

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5780, #6334

**Comment by emyl3** [comment]
Findings:
- `/patients` - multiple patients [dev.simplereport.gov-Patients-multiple-2025-03-03.json](https://github.com/user-attachments/files/19056103/dev.simplereport.gov-Patients-multiple-2025-03-03.json)

**Comment by emyl3** [comment]
Ready for review
